dotcom-rendering/src/web/components/ArticlePage.tsx

```diff
@@ -55,6 +57,11 @@ export const ArticlePage = (props: WebProps | AppProps) => {
 				`}
 			/>
 			<SkipTo id="maincontent" label="Skip to main content" />
+			<SkipTo id="navigation" label="Skip to navigation" />
+			<Lightbox format={format} images={article.imagesForLightbox} />
+			<Island clientOnly={true}>
+				<LightboxHash />
+			</Island>
```

Comment:
Could we add this island optionally based on whether there are images for lightbox or not?

Reply:
👍  [Fixed](https://github.com/guardian/dotcom-rendering/pull/7129/commits/b228dda5d3f01a55955247b9c1e778171233b22a)